crates/torii/core/src/sql.rs

```diff
@@ -193,6 +204,9 @@ impl Sql {
                                ?, ?, ?) ON CONFLICT(id) DO UPDATE SET \
                                updated_at=CURRENT_TIMESTAMP, executed_at=EXCLUDED.executed_at, \
                                event_id=EXCLUDED.event_id RETURNING *";
+        // if timeout doesn't work
+        // fetch to get entity
+        // if not available, insert into queue
         let mut entity_updated: EntityUpdated = sqlx::query_as(insert_entities)
```

Comment:
Is there a reason as to why we push our broker message to the queue? Because the insert_entities query is executed without the queue. Are we going at some point insrt it into queue in some cases?

Reply:
i have updated it to use queue in the next commit on stack